R/derive_var_atoxgr.R

```diff
@@ -151,7 +151,7 @@ derive_var_atoxgr_dir <- function(dataset,
   # Check metadata data set has required variables
   assert_data_frame(
     meta_criteria,
-    required_vars = exprs(TERM, GRADE_CRITERIA_CODE, FILTER, DIRECTION, SI_UNIT_CHECK, VAR_CHECK)
+    required_vars = exprs(TERM, GRADE_CRITERIA_CODE, FILTER, DIRECTION, UNIT_CHECK, VAR_CHECK)
```

Comment:
umm yeah i am little worried about this impacting users?  but maybe not following how this function is working